styles/Nav.module.css

```diff
@@ -0,0 +1,18 @@
+.nav {
+  height: 50px;
+  padding: 10px;
+  background: #000;
+  color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+}
+.nav ul {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style: none;
+}
+.nav ul li a {
+  margin: 5px 15px;
+}
```

Comment:
its better to create classes for the `ul` and `a` elements than to style through the tag names

Reply:
Ok fixed